packages/filter.punches/spec/filterBehavior.ts

```diff
@@ -5,14 +5,18 @@ import {
 
 import { filters } from '../dist'
 
+
+declare var ko : any
```

Comment:
`: KnockoutStatic`?

Reply:
okay, I defined `export type KnockoutStatic = any` in `options.ts` an use it in some known cases, see https://github.com/Auge19/tko/pull/4/commits/fa3b0599dcc7dee1514cd9a0c27a5878b8da9bf6